### PR TITLE
Issue622 Fix

### PR DESCRIPTION
**Pull request by maestretti**
Fixing a JSON key error for #622

### CODE REVIEW

File: shared/iam_audit.py [hunk 1]
```diff
@@ -455,7 +455,7 @@ def find_admins_in_account(
                         Finding(
                             region,
                             "IAM_LINTER",
-                            policy["Arn"],
+                            role["Arn"],
```
Comment (0xdabbad00) [inline shared/iam_audit.py]: I think this one should be `user`

Comment (maestretti) [reply shared/iam_audit.py]: Agreed. Updated the branch.

**Comment by 0xdabbad00** [comment]
@maestretti Are you able to sign the CLA? I know we had some issues in the past with it, but I don't know if those have been resolved yet.

**Comment by maestretti** [comment]
@0xdabbad00 I signed it... looks like an email mismatch on the commit and my GitHub account which I fixed and re-checked, but it doesn't seem to see the match (yet?).

**Comment by maestretti** [comment]
@0xdabbad00 this looks ready to go?

**Comment by 0xdabbad00** [comment]
Yes, thank you for following up and for submitting the PR.

**Pull request merged**